waku.md

```diff
@@ -322,16 +328,67 @@ Waku is a different subprotocol from Whisper so it isn't directly compatible. Ho
 
 It is desirable to have a strategy for maintaining forward compatibility between `waku/0` and future version of waku. Here we outline some concerns and strategy for this.
 
-
-## Security considerations
+## Appendix A: Security considerations
 
 There are several security considerations to take into account when running Waku. Chief among them are: scalability, DDoS-resistance and privacy. These also vary depending on what capabilities are used, such as mailserver, light node, and so on.
 
-### Light node privacy
+### Scalability and UX
+
+**Bandwidth usage:**
+
+In version 0 of Waku, bandwidth usage is likely to be an issue. For more investigation into this, see the theoretical scaling model described [here](https://vac.dev/fixing-whisper-with-waku).
+
+**Mailserver High Availability requirement:**
+
+A mailserver has to be online to receive messages for other nodes, this puts a high availability requirement on it. This can be somewhat mitigated by the use of something like [MVDS](https://specs.vac.dev/mvds.html) on top, thereby treating a mailserver as a form of cache.
```

Comment:
I am not sure I would mention MVDS, currently in its state it does not address the issue of offline messaging, especially for one-to-one communication (no third party relaying of one-to-one messages, nor a plan to add support for it), and it's not clear that it will solve this problem in the future, it might somewhat help with public chats although currently we are not using to relay messages.

Reply:
(see Discord)

Reply:
removed mention and turned HA to availability

Reply:
removed mvds refer